walk-in-interview/src/main/java/com/google/utils/ServletUtils.java

```diff
@@ -102,6 +103,22 @@ public static Date getDateParameter(HttpServletRequest request,
         }
     }
 
+    /**
+     * @return the value of parameter with the {@code name} in the {@code request}
```

Comment:
is there no need to write @param in the docs? same comment on docs for other functions

Reply:
this format is also allowed